test/e2e/load.go

```diff
@@ -117,20 +119,24 @@ var _ = framework.KubeDescribe("Load capacity", func() {
 		podsPerNode int
 		image       string
 		command     []string
+		// What kind of resource we want to create
+		kind schema.GroupKind
 	}
 
 	loadTests := []Load{
 		// The container will consume 1 cpu and 512mb of memory.
-		{podsPerNode: 3, image: "jess/stress", command: []string{"stress", "-c", "1", "-m", "2"}},
-		{podsPerNode: 30, image: "gcr.io/google_containers/serve_hostname:v1.4"},
+		{podsPerNode: 3, image: "jess/stress", command: []string{"stress", "-c", "1", "-m", "2"}, kind: api.Kind("ReplicationController")},
+		{podsPerNode: 30, image: "gcr.io/google_containers/serve_hostname:v1.4", kind: api.Kind("ReplicationController")},
```

Comment:
I wouldn't change predefined tests. We should have a test that has everything inside, not 3 different tests. [Obviously not in this PR].